src/shared/XLinkDispatcher.c

```diff
@@ -485,7 +485,11 @@ int DispatcherUnblockEvent(eventId_t id, xLinkEventType_t type, streamId_t strea
 int pthread_t_compare(pthread_t a, pthread_t b)
 {
 #if (defined(_WIN32) || defined(_WIN64) )
+#ifdef __GNUC__
```

Comment:
Candidate for moving to win_pthread.c/h to avoid conditional implementation?

Reply:
Indeed - along with many others hah:D
XLink has some rough edges in this regards - if'd be written in C++, would probably be quite smaller and more concise as a whole. 
Some changes are incoming in this regard, specifically to USB part

Reply:
But this looks okay for now